lxd/auth/oidc/oidc.go

```diff
@@ -414,7 +414,19 @@ func (o *Verifier) Callback(w http.ResponseWriter, r *http.Request) {
 	}
 
 	handler := rp.CodeExchangeHandler(func(w http.ResponseWriter, r *http.Request, tokens *oidc.Tokens[*oidc.IDTokenClaims], state string, rp rp.RelyingParty) {
-		err := o.startSession(r.Context(), w, tokens.IDToken, tokens.RefreshToken)
+		userInfo, err := o.userInfo(r.Context(), tokens.AccessToken)
```

Comment:
could use an anonymous function here to only have a single error handler, e.g.

```go
err := func() error {
userInfo, err := o.userInfo(r.Context(), tokens.AccessToken)
...
res, err := o.getResultFromClaims(userInfo, userInfo.Claims)
...
}
		if err != nil {
			_ = response.ErrorResponse(http.StatusInternalServerError, fmt.Errorf("Failed to start a new session: %w", err).Error()).Render(w, r)
			return
		}
```

Reply:
Done